test/controllers/deploys_controller_test.rb

```diff
@@ -26,23 +26,15 @@
 
   as_a_viewer do
     describe "a GET to :index" do
-      setup { get :index, project_id: project.to_param, format: format }
```

Comment:
drive by cleanup since I initially wanted to add it into deploys controller, but that turned out to be messy